sopel/builtins/wikipedia.py

```diff
@@ -327,12 +327,22 @@ def mw_image_description(server, image):
 def mw_info(bot, trigger, match=None):
     """Retrieves and outputs a snippet of the linked page."""
     server = match.group(1)
-    page_info = urlparse(match.group(2))
-    article = unquote(page_info.path)
+    page_info = urlparse(match.match.string)
```

Comment:
We should try to get away from using the `match` argument. The behavior of `trigger` is now identical, because the `match` is also the `trigger`:

https://github.com/sopel-irc/sopel/blob/7693af3356f59be7c825cafd91251dc88f5caf2a/sopel/plugins/rules.py#L1778-L1792

Don't think we've gotten around to formally deprecating `match`, but we should. Maybe in 8.1.